### PR TITLE
DOCS Clarified file permission control

### PR DESCRIPTION
**Pull request by chillu**

### CODE REVIEW

File: docs/en/02_Developer_Guides/14_Files/03_File_Security.md [hunk 1]
```diff
@@ -2,13 +2,85 @@ summary: Manage access permission to assets
...
+Access to create or delete files generally aligns with the edit access described above.
+
+Note that even if the permissions above allow access,
+you need to have access to a mechanism to view or edit file information. 
+Most commonly this is through the "Access to Files section" permission.
+Custom implementations (e.g. APIs or custom file viewers) can have
+further restrictions in your project. 
+
+<div class="warning" markdown="1">
+When implenting your own `canView()` logic through [extensions](/developer_guides/extending/extensions),
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/14_Files/03_File_Security.md]: FYI I don't think these HTML sections get parsed as markdown right now, so this link would get rendered literally

Comment (chillu) [reply docs/en/02_Developer_Guides/14_Files/03_File_Security.md]: We have lots of other sections like that in the docs: https://github.com/silverstripe/silverstripe-framework/search?q=%3Cdiv+class%3D%22warning%22+markdown%3D%221%22%3E&unscoped_q=%3Cdiv+class%3D%22warning%22+markdown%3D%221%22%3E. They render just fine: https://docs.silverstripe.org/en/4/contributing/documentation

Comment (chillu) [reply docs/en/02_Developer_Guides/14_Files/03_File_Security.md]: Oh wait, you mean the *link* within that ... works fine as well: https://docs.silverstripe.org/en/4/developer_guides/model/versioning (see "The extension is automatically applied...")

**Comment by chillu** [comment]
> There's some inconsistency with using periods at the end of bullet pointed sentences, but the content LGTM

OK I've consistently added periods

File: docs/en/02_Developer_Guides/14_Files/03_File_Security.md (hunk 1 above)
Comment (robbieaverill) [reply docs/en/02_Developer_Guides/14_Files/03_File_Security.md]: OK, sweet!

**Pull request merged**